pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
```

Comment:
_"high-throughput sequencing"_ is a commonly used term for this (often abbreviated to HTS), so those words go together quite nicely. I added _(genomics)_ to help clarify. If I'm honest I can't think of any alternatives that sound better to my ear..